charts/base-cluster/templates/descheduler/descheduler.yaml

```
@@ -1,22 +1,18 @@
-{{- if .Values.descheduler.enabled -}}
-{{- $kubeMinorVersion := .Capabilities.KubeVersion.Minor -}}
-{{- $versionMatrix := dict -}}
-{{- $latestVersion := "" -}}
+{{- $selectedVersion := "" -}}
 {{- with .Values.global.helmRepositories.descheduler.charts -}}
-  {{- $versionMatrix = dict 27 .descheduler_27 28 .descheduler_28 29 .descheduler_29 -}}
-  {{- $latestVersion = .descheduler -}}
+  {{- with (index . (printf "descheduler 0.%d.x" ($.Capabilities.KubeVersion.Minor | int))) -}}
+    {{- $selectedVersion = . -}}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 0 but found 4

(indentation)